contrib/cloud.google.com/go/pubsub.v1/pubsub.go

```diff
@@ -63,15 +64,33 @@ func (r *PublishResult) Get(ctx context.Context) (string, error) {
 	return serverID, err
 }
 
+type config struct {
+	// If set, overrides the trace's service name tag.
+	serviceName string
+}
+
+// A ReceiveHandlerOption is used to customize spans started by WrapReceiveHandler.
+type ReceiveHandlerOption func(cfg *config)
```

Comment:
```suggestion
type Option func(cfg *config)
```

Reply:
Sure, I can do this. My motivation for this verbosity: We might need to add options to `Publish` in the future.

Reply:
I understand. That makes sense. In that case how about `ReceiveOption` and `PublishOption`?

Reply:
OK, I renamed `Option` to `ReceiveOption`.